gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/db/FabricDatabase.java

```diff
@@ -244,7 +244,11 @@ Licensed to the Apache Software Foundation (ASF) under one or more
   /** to allow for initial DDL replay even with failures */
   private final boolean allowBootWithFailures = Boolean.getBoolean(
       com.pivotal.gemfirexd.Property.DDLREPLAY_ALLOW_RESTART_WITH_ERRORS);
-  
+
+  /** to allow skipping of index sanity check on restart */
+  public static boolean skipIndexCheck = Boolean.getBoolean(
+      com.pivotal.gemfirexd.Property.DDLREPLAY_NO_INDEX_CHECK);
+
   /**
```

Comment:
Though it works as is, the property names should also remove the "gemfirexd." prefix since that will be handled automatically by SystemProperties. That was the idea that if we were to change to a new product name (snappydata now) then system properties will automatically switch to using that prefix (in addition to older one). Also the Gem layer properties should also use the same product name for consistency.